sections/futures/MarketDetails/MarketDetails.tsx

```diff
@@ -136,7 +108,7 @@ const MarketDetails: React.FC<MarketDetailsProps> = ({ mobile }) => {
 						{children}
 					</TimerTooltip>
 				);
-			case fundingTitle:
+			case 'Inst. Funding Rate' || '1H Funding Rate':
```

Comment:
Is there something else we can use here other than label strings as this would break on other languages?

Reply:
The issue is that we use the titles as keys here, but the title is conditional. All of these keys are based on the title that is shown, so we would need to update the entire component to display translations.

Reply:
Let me write a separate issue for that and resolve it separately from this PR.

Reply:
ok sounds good

Reply:
Issue here: https://github.com/Kwenta/kwenta/issues/1207